src/Client.php

```diff
@@ -49,6 +49,7 @@
  * The Google API Client
  * https://github.com/google/google-api-php-client
  */
+#[\AllowDynamicProperties]
```

Comment:
Why are we adding dynamic properties to this class? This class is not extended by anything, nor does it extend anything. So any dynamic properties it has should be defined.

Are you seeing dynamic property warnings anywhere for this class?

Reply:
You are right, removing this change.